src/qt/res/css/light-hires.css

```diff
@@ -1,5 +1,5 @@
 WalletFrame {
-border-image: url(':/images/light/drkblue_walletFrame_bg') 0 0 0 0 stretch stretch;
+border-image: url(':/images/light-retro/drkblue_walletFrame_bg') 0 0 0 0 stretch stretch;
```

Comment:
Should not update this file, should copy `light-hires` theme to `light-hires-retro` and update it instead. Pls revert all changes made below.

Reply:
@UdjinM6 Previous to this PR, the `light-hires.css` file actually makes reference to `light` theme. Since we re-named `light` to `light-retro`, I believe the references in `light-hires.css` should be updated to refer to the new name, right?

In that case I believe this should stand. Otherwise `light-hires.css` is referencing the new, updated images.

Reply:
Hmm, no... Let me explain how I see it should be done, step by step:
1. `light-hires` should be copied to `light-hires-retro`;
2. `light-hires-retro` should have all links updated from `light` to `light-retro`, should have no other code changes;
3. `light-hires` should have all images updated to ones with the new logo, should have no code changes at all.